tests/plugins/preview/manual/previewcdn.html

```diff
@@ -0,0 +1,72 @@
+<div id="editor"></div>
+<div id="loading_box" style="margin: 20px 0"></div>
+<div id="editor_cdn">
+	<h1>Sample <span style="color: red;">document</span></h1>
+	<p>Lorem ipsum</p>
+	<div style="page-break-after:always">
+		<span style="display:none">&nbsp;</span>
+	</div>
+	<p>dolor sit amet.</p>
+</div>
+
+<script>
+	// 1. Load local editor by default with bender.
+	// 2. Remove existing instance and add script with editor from CDN.
+	// 3. Wait for the new editor to be loaded from CDN and initialize it.
+	// 4. Add icons in IE <11 from local.
+	var path = 'https://raw.githack.com/ckeditor/ckeditor4/903fa05e6bde756a7f4e618b6d24b4db21da5ab7';
+
+	CKEDITOR.replace( 'editor' );
+	CKEDITOR.once( 'instanceReady', function() {
+		var scripts = document.querySelectorAll( 'script' ),
+			head = CKEDITOR.document.getHead();
+
+		CKEDITOR.tools.array.forEach( scripts, function( script ){
+			script.parentNode.removeChild( script );
+		} );
+
+		CKEDITOR.instances.editor.destroy();
+		window.CKEDITOR = null;
+
+		var newScript = document.createElement( 'script' ),
+			loadingBox = document.getElementById( 'loading_box' ),
+			loadingText = 'loading editor from CDN';
+
+		newScript.type = 'text/javascript';
+		newScript.src = path + '/ckeditor.js';
+		document.head.appendChild( newScript );
+
+		var interval = setInterval( function() {
+			try {
+				if( CKEDITOR.status === 'loaded' ){
+					clearInterval( interval );
+
+					CKEDITOR.replace( 'editor_cdn', { plugins: 'wysiwygarea, print, pagebreak, toolbar, clipboard, image, floatingspace, link, basicstyles' } );
```

Comment:
This is a bit problematic since we have basic, standard and full presets and manual tests run for such preset use default plugins. So for `full all` used during testing phase we would like to have all plugins which are default for this preset.

The solution could be reading `plugins` list from the initial editor instance (before destroying) and then pass it to new editor config. WDYT?

Reply:
Good point. I didn't think about presets and your idea might work. Thanks!